src/data/real/nnreal.lean

```diff
@@ -549,8 +549,7 @@ by simpa only [div_eq_mul_inv] using mul_pos hr (inv_pos.2 hp)
 
 protected lemma mul_inv {r p : ℝ≥0} : (r * p)⁻¹ = p⁻¹ * r⁻¹ := nnreal.eq $ mul_inv_rev₀ _ _
 
-lemma div_self_le (r : ℝ≥0) : r / r ≤ 1 :=
-if h : r = 0 then by simp [h] else by rw [div_self h]
+lemma div_self_le (r : ℝ≥0) : r / r ≤ 1 := div_self_le_one (r : ℝ)
```

Comment:
Do we still need this version now that we have the generalisation?

Reply:
Yes, `nnreal` is not a `linear_ordered_field`.

Reply:
Thanks, I get the point now.